Packages/Web/lib/propertyQuerries.flp

```diff
@@ -20,7 +20,6 @@ func propertyIsForTransform(_ property)
   return
```

Comment:
We should prob change the name of this function now since were removing the rotation property out. Let's call this function: `func propertyIsNotAnimated(_ property)

Reply:
can also add a comment here explaining that were choosing to not animate scale and anchor point because we can't animate the different components of the transform independently using WAAPI